core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -793,7 +799,14 @@ public function cutAll()
 		{
 			foreach ($arrClipboard[$this->strTable]['id'] as $id)
 			{
-				$this->cut($id); // do not urldecode() here (see #6840)
+				try
+				{
+					$this->cut($id); // do not urldecode() here (see #6840)
+				}
+				catch (AccessDeniedException)
+				{
+					// noop
+				}
```

Comment:
Why do we catch the exception here?

Reply:
We catch it in all places where multiple operations are happening. If you cut 30 IDs and you have access to 25, then the 25 should work and the other ones should be ignored.